### PR TITLE
include symbols and src from referenced projects

### PR DESCRIPTION
**Pull request by pms1969**
* include source files for referenced projects in symbols nupkg's
* ensure pdb's for referenced projects are included in nupkg's
* add command line option for pack (include-referenced-projects)
* create some tests for some of the above

### CODE REVIEW

File: tests/Paket.Tests/ProjectFile/InterProjectDependencySpecs.fs [hunk 1]
```diff
@@ -37,8 +37,8 @@ let ``should detect relative path for dependencies in Project2 proj file``() =
         ProjectFile.TryLoad("./ProjectFile/TestData/Project2.fsprojtest").Value.GetInterProjectDependencies()
         |> List.map (fun p -> p.RelativePath)
 
-    paths.[0] |> shouldEqual "..\\..\\src\\Paket\\Paket.fsproj"
-    paths.[1] |> shouldEqual "..\\Paket.Core\\Paket.Core.fsproj"
+    paths.[0] |> shouldEqual "..\\..\\..\\..\\src\\Paket\\Paket.fsproj"
+    paths.[1] |> shouldEqual "..\\..\\..\\..\\src\\Paket.Core\\Paket.Core.fsproj"
```
Comment (forki) [inline tests/Paket.Tests/ProjectFile/InterProjectDependencySpecs.fs]: why did this change?

Comment (pms1969) [inline tests/Paket.Tests/ProjectFile/InterProjectDependencySpecs.fs]: The unit tests used it.  From where it is called by nunit, those projects do not exist in the paths that were specified.

**Comment by forki** [comment]
cool

**Comment by forki** [comment]
Sorry had to revert. Something is wrong with the paths. We I try to release Paket can't pack iteself anymore.

Please resubmit the PR and try to run "build.cmd NuGet" on the repo.

**Comment by pms1969** [comment]
ok, Probably be tomorrow now.  I'll see what the problem is and fix.

**Comment by forki** [comment]
Thanks.

**Pull request merged**